src/components/LiveExample.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react'
+import { convertToReactComponent } from '@patternfly/ast-helpers'
+import * as reactCoreModule from '@patternfly/react-core'
```

Comment:
I wonder if importing the whole module is going to impact performance at all. Opposed to importing things based on the md's list?

Reply:
I _think_ it should be ok as this is how the docs framework currently does it in example.js too, but I do share some of the same concern.

One advantage of this route is that we actually shouldn't need to list imports in the md files anymore, but if there is a notable performance impact we could refactor this to something that does require listing imports in the md, or maybe come up with a way of analyzing the imports of the example files themselves 🤔